src/components/JfDrawerComponent.spec.js

```diff
@@ -0,0 +1,28 @@
+import {shallowMount} from '@vue/test-utils';
+import JfDrawerComponent from './JfDrawerComponent.vue';
+
+describe('JfDrawerComponent', () => {
+    it('should set open variable to be true', () => {
+        const wrapper = shallowMount(JfDrawerComponent, {
+            propsData: {
+                openFirst: '0'
+            },
+            stubs: ['b-collapse', 'b-card']
+        });
+        expect(wrapper.vm.opened).toBe(true);
+    });
+});
+
+
+describe('JfDrawerComponent', () => {
```

Comment:
Pls unify the describes as usually single describe is used in order to test single component. You can use nested describe to group few test with common concept and setup functions.